admin_manual/configuration/server/config_apps_sample_php_parameters.rst

```diff
@@ -0,0 +1,124 @@
+==========================
+Apps Config.php Parameters
+==========================
+
+This document describes parameters for apps maintained by ownCloud and are
+not part of the core system.
+
+.. note:: All keys are only valid if the corresponding app is installed and enabled.
+   You MUST copy the keys needed to the active config.php file.
```

Comment:
I'd suggest it's better to use [an admonition](http://docutils.sourceforge.net/docs/ref/rst/directives.html#admonitions) than all-caps.